extensions/tekton/generator-v1alpha1/go.mod

```diff
@@ -2,10 +2,17 @@ module github.com/fabric8io/kubernetes-client/extensions/tekton/generator
 
 require (
 	github.com/fabric8io/kubernetes-client/generator v0.0.0
-	github.com/tektoncd/pipeline v0.10.2
```

Comment:
as mentioned, v1alpha1 should not be updated to the latest version!

Reply:
Yup I misunderstood it. I thought the v1beta1 client for only v1beta1 resources. I think it has both latest v1alpha1 and v1beta1. While v1alpha client is for old v1alpha1 resources only. In that case, we don't need to update v1alpha1 client then